src/modules/shopgate/helpers/basket.php

```diff
@@ -204,6 +206,20 @@ protected function loadOrderBasketPayment(shopgate_oxbasket &$oxBasket, Shopgate
     }
 
     /**
+     * @param shopgate_oxbasket $oxBasket
+     * @param ShopgateCartBase  $shopgateOrder
+     */
+    protected function loadOrderBasketTotal(shopgate_oxbasket &$oxBasket, ShopgateCartBase $shopgateOrder)
+    {
+        $oxPrice = oxNew('oxPrice');
+        $oxPrice->setBruttoPriceMode();
+        $fDelVATPercent = $oxBasket->getProductsPrice()->getMostUsedVatPercent();
+        $oxPrice->setVat($fDelVATPercent);
```

Comment:
You don't need to put this expression in a variable: $oxBasket->getProductsPrice()->getMostUsedVatPercent()
Just pass it directly along:

        $oxPrice->setVat($oxBasket->getProductsPrice()->getMostUsedVatPercent());